lazy_relay.js

```diff
@@ -523,6 +582,22 @@ function _observeCallReqContFrame(frame) {
 };
 
 function LazyRelayOutReq(conn, inreq) {
+    this.channel = null;
+    this.conn = null;
+    this.start = null;
+    this.remoteAddr = null;
+    this.logger = null;
+    this.inreq = null;
+    this.id = null;
+    this.serviceName = null;
+    this.callerName = null;
+    this.timeout = null;
+    this.operations = null;
+    this.timeHeapHandle = null;
+    this.alive = null;
```

Comment:
Setting everything to `null` is more of a safety thing -- I want to induce reference errors when we accidentally use an object while it's still part of the free list